paper.bib

```diff
@@ -19063,6 +19063,151 @@ @Article{Thrampoulidis2015LASSO
+  Author                   = {Liu, S. and Suzuki, T. and Relator, R. and Sese, J. and Sugiyama, M. and Fukumizu, K.},
+  Journal                  = {Annals of Statistics},
+  Year                     = {2017},
+
+  Month                    = {06},
+  Number                   = {3},
+  Pages                    = {959--990},
+  Volume                   = {45}
+}
+
+@Book{Mackay2003,
+  Title                    = {Information theory, inference and learning algorithms},
+  Author                   = {MacKay, D.},
+  Publisher                = {Cambridge university press},
+  Year                     = {2003}
+}
+
+@Article{Shimodaira2000,
+  Title                    = {Improving predictive inference under covariate shift by weighting the log-likelihood function},
+  Author                   = {Shimodaira, A.},
+  Journal                  = {Journal of Statistical Planning and Inference},
+  Year                     = {2000},
+  Number                   = {2},
+  Pages                    = {227 - 244},
+  Volume                   = {90},
+
+  Doi                      = {https://doi.org/10.1016/S0378-3758(00)00115-4},
+  ISSN                     = {0378-3758},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0378375800001154}
+}
+
+@Article{Sugiyama2008,
+  Title                    = {Direct Importance Estimation for Covariate Shift Adaptation},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Nakajima, S. and Kashima, H. and von B\"unau, P. and Kawanabe, M.},
+  Journal                  = {Annals of the Institute of Statistical Mathematics},
+  Year                     = {2008},
+  Number                   = {4},
+  Pages                    = {699--746},
+  Volume                   = {60}
+}
+
+@Article{SUGIYAMA2011735,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {M. Sugiyama and T. Suzuki and Y. Itoh and T. Kanamori and M. Kimura},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24},
+
+  Doi                      = {https://doi.org/10.1016/j.neunet.2011.04.003},
+  ISSN                     = {0893-6080},
+  Keywords                 = {Two-sample test, Homogeneity test, Density ratio estimation, Unconstrained least-squares importance fitting, Pearson divergence},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0893608011001262}
+}
+
+@Article{Sugiyama2011,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Itoh, Y. and Kanamori T. and Kimura M.},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24}
+}
+
+@Book{Sugiyama2012,
```

Comment:
@Book{Sugiyama2012Density,
  title      = {Density ratio estimation in machine learning},
  publisher  = {Cambridge University Press, Cambridge},
  year       = {2012},
  author     = {Sugiyama, Masashi and Suzuki, Taiji and Kanamori, Takafumi},
  isbn       = {978-0-521-19017-6},
  note       = {With a foreword by Thomas G. Dietterich},
  doi        = {10.1017/CBO9781139035613},
  mrclass    = {62G05 (62G20 62H30 68-02 68T05)},
  mrnumber   = {2895762},
  mrreviewer = {Pierre Alquier},
  pages      = {xii+329},
  url        = {https://doi.org/10.1017/CBO9781139035613},
}